app/models/elastic_custom_client.rb

```diff
@@ -0,0 +1,19 @@
+require 'multi_json'
+require 'faraday'
+require 'elasticsearch/api'
+
+class ElasticCustomClient
+  include Elasticsearch::API
+
+  CONNECTION = ::Faraday::Connection.new url: ENV["ELASTICSEARCH_URL"]
+
+  def perform_request(method, path, params, body)
+    puts "--> #{method.upcase} #{path} #{params} #{body}"
+
+    CONNECTION.run_request \
+      method.downcase.to_sym,
+      path,
+      ( body ? MultiJson.dump(body): nil ),
+      {'Content-Type' => 'application/json'}
```

Comment:
Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.